lib/kitchen/directions/bake_free_response/v1.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeFreeResponse
+  class V1
+    renderable
+
+    def bake(chapter:, metadata_source:, append_to:)
+      @metadata_elements = metadata_source.children_to_keep.copy
+
+      @free_response_questions = []
+      free_response_clipboard = Kitchen::Clipboard.new
+      chapter.non_introduction_pages.each do |page|
+        free_response_questions = page.free_response
+        next if free_response_questions.none?
+
+        free_response_questions.search('h3').trash
+        title = Kitchen::Directions::EocSectionTitleLinkSnippet.v1(page: page)
+        free_response_questions.each do |free_response_question|
+          free_response_question.prepend(child: title)
+          free_response_question&.cut(to: free_response_clipboard)
+        end
+        @free_response_questions.push(free_response_clipboard.paste)
+        free_response_clipboard.clear
+      end
```

Comment:
A clipboard is essentially an array, so I'm wondering if we don't need both an array and a clipboard on lines 10 and 11.

```ruby
def bake(chapter:, metadata_source:, append_to:)
  @metadata_elements = metadata_source.children_to_keep.copy

  @free_response_clipboard = Kitchen::Clipboard.new
  chapter.non_introduction_pages.each do |page|
    free_response_questions = page.free_response
    next if free_response_questions.none?

    free_response_questions.search('h3').trash
    title = Kitchen::Directions::EocSectionTitleLinkSnippet.v1(page: page)
    free_response_questions.each do |free_response_question|
      free_response_question.prepend(child: title)
      free_response_question&.cut(to: @free_response_clipboard)
    end
  end

  append_to_element = append_to || chapter
  @title_tag = append_to ? 'h3' : 'h2'

  append_to_element.append(child: render(file: 'free_response.xhtml.erb'))
end
```

and then do `@free_response_clipboard.paste` in the `.xhtml.erb` file?